solr/core/src/test/org/apache/solr/update/TestInPlaceUpdatesDistrib.java

```diff
@@ -1489,6 +1489,12 @@ private String getBaseUrl(String id) {
     return slice.getLeader().getCoreUrl();
   }
 
+  protected String getBaseUrl(HttpSolrClient client) {
+    return client
+        .getBaseURL()
+        .substring(0, client.getBaseURL().length() - DEFAULT_COLLECTION.length() - 1);
+  }
```

Comment:
add a comment to say that we assume that client's URL ends with `/collection1`.
But you still have client.getBaseURL which we're trying to get rid of.  So...

Reply:
adding a comment..   and someday, when we eliminate HttpSolrClient, will have to properly figure out how to get the underlying Jetty server...